### PR TITLE
code to insert timestamp in bigquery table

### PR DESCRIPTION
**Pull request by mivelRBFA**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+venv\
```
Comment (GiunioRBFA) [inline .gitignore]: try to use a template one.

check her for example https://github.com/github/gitignore

I think it should be even proposed by github when you start the project

File: main.py [hunk 2]
```diff
@@ -13,6 +11,19 @@ async def read_root():
...
 
 
+#code to insert timestamp into bigquery table
+@app.get("/bq_insert_timestamp")
+async def read_root():
+    now = datetime.now()
+    my_dict = {'Timestamp': str(now)}
+    row_to_insert = [my_dict]
+    bq_client = bigquery.Client()
+    table = bq_client.get_table("{}.{}.{}".format("rbfa-workshop-sandboxes", "timestamps_milan", "timestamps"))
```
Comment (GiunioRBFA) [inline main.py:21]: nice to use format, but it has to be a specific case.

In this one the f-string formatting is optimal

File: .gitignore (hunk 1 above)
Comment (mivelRBFA) [reply .gitignore]: Thank you,, it's done

File: main.py (hunk 2 above)
Comment (mivelRBFA) [reply main.py:21]: done

**Pull request merged**